Glossary.md

```diff
@@ -0,0 +1,149 @@
+
+## Application Accounts
+**Application Accounts:** Used by developers and users interacting with applications deployed on Hedera. These accounts enable users to interact with the functionality provided by dApps, such as gaming, finance, or other decentralized services.
+
+## Auto Associations
+**Auto Associations**: The ability of accounts to automatically associate with tokens. An auto association slot is one or more slots you approve that allow tokens to be sent to your contract without explicit authorization for each token type. If this property is not set, you must approve each token before it is transferred to the contract for the transfer to be successful via the `TokenAssociateTransaction` in the SDKs.
+
+## Auto-Renew Accounts
+**Auto-Renew Accounts**: A feature designed to automatically renew the lifecycle of certain entities (like accounts, files, smart contracts, topics, or tokens) by funding their renewal fees. This feature is particularly useful for entities that require continuous operation over long periods, as it automates the process of keeping these entities active by periodically charging a linked auto-renew account.
+
+---
+
+## BLS
+**Boneh-Lynn-Shacham (BLS)**: A paired-elliptic-curve based signature algorithm with attractive characteristics for aggregate signatures (signatures produced by aggregating many individual signatures, and verifiable with a single public key).  BLS comes from the names of the authors of the initial research paper (Dan Boneh, Ben Lynn, Hovav Shacham).
+
+## Block Proof
+**Block Proof**: Cryptographic proof that a particular block contains a given block item. Block proofs will also be the basis of State Proofs as the root hash of states will be included in each block. see
+
+## Block Signature
+**Block Signature**: Aggregated (BLS) signature of the hash of Block stream merkle root. Appended to the end of a block stream it is used to verify the block was produced by the majority weight of the network. These signatures will also be used for Invalid State Detection.
+
+## Block Stream
+**Block Stream**: A fully verifiable stream of data consisting of individual blocks corresponding to consensus rounds. Each block contains all events, transactions, and state changes that occurred within one consensus round. A block is transmitted from consensus node to block nodes as a stream of individual block items, which form a merkle tree that is verified by a Block Signature. Each block is connected to the previous block by a Block Hash which, together form a block chain.
+
+---
+
+
+## Civilian Accounts
+**Civilian Accounts**:  User accounts that are not associated with any special roles or permissions beyond standard user functionality. By contrast, System Accounts and Node Accounts are specialized accounts that have specific roles and responsibilities within the infrastructure and governance of the Hedera network.
```

Comment:
Note, maybe, that term seems to be confined to tests?  I've certainly never heard it used in any other context.

Reply:
Also, the Node Account or System Account are not "specialized", the accounts are the same as any other account, we just _treat_ them differently based on configuration values (System Accounts) and address book information (Node Accounts).

Note also that a ContractID works like an account identifier (and _is_ an account ID) because a smart contract is an account with certain fields set to contract-specific values.